server/src/router.ts

```diff
@@ -0,0 +1,9 @@
+import { Router } from 'express';
+
+// eslint-disable-next-line sort-imports
+import film from 'film';
```

Comment:
Why is this import not at the top with the other one?

Reply:
A personal habit to have imports representing entities separate from other imports.